Challenge-1/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "forta-bots-challenge",
+  "displayName": "Forta Bots Challenge-1",
+  "version": "0.0.1",
+  "description": "Forta bot that detects creation and updation of other Forta bots on a network",
+  "repository": "https://github.com/yashthipsay/Forta_Bots_Challenges/tree/%40intern-challenges/one",
+  "chainIds": [
+    1,
+    137,
+    10,
+    56,
+    250,
+    42161,
+    43114
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "alert": "npm run build && forta-agent run --alert",
+    "sequence": "npm run build && forta-agent run --sequence",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "stake": "forta-agent stake",
+    "test": "jest"
+  },
+  "dependencies": {
+    "ethers": "^6.13.0",
+    "forta-agent": "^0.1.48",
+    "forta-agent-tools": "^3.2.12"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.10",
+    "@types/nodemon": "^1.19.0",
+    "jest": "^29.7.0",
+    "nodemon": "^2.0.8",
+    "ts-jest": "^29.1.1",
+    "typescript": "^4.3.4"
+  }
```

Comment:
`prettier` has to be installed and the code needs to be formatted.

You can check [here](https://www.notion.so/nethermind/Bot-Readiness-Checklist-fdfe8522762b4242955a881e4be944d6?pvs=4#09a5e69f8b324cd1ae29fcd96cacdcf1) for instructions